openshift-hack/cmd/k8s-tests-ext/k8s-tests.go

```diff
@@ -160,6 +160,7 @@ func withExcludedTestsFilter(baseExpr string) string {
 		"[Slow]",
 		"[Flaky]",
 		"[Local]",
+		"[Feature:HPA]", // We enable these tests through its own HPA prow job
```

Comment:
@smg247 could you help review this change, please? I have the impression this filter is not meant to be used by features. Does the framework have a pattern to accommodate what we're trying to do with this change (skip HPA tests by default, but run them in its own job)?